src/node/desktop/src/main/application-launch.ts

```diff
@@ -70,19 +71,18 @@ export class ApplicationLaunch {
 
     // resolve working directory
     const workingDir = options.workingDirectory ?? path.dirname(options.projectFilePath || '');
-    setenv('RS_INITIAL_WD', workingDir);
+    setenv(kRStudioInitialWorkingDir, workingDir);
 
     // resolve project file, if any
     const projectFile = options.projectFilePath ?? resolveProjectFile(workingDir);
     if (existsSync(projectFile)) {
-      setenv('RS_INITIAL_PROJECT', projectFile);
+      setenv(kRStudioInitialProject, projectFile);
     }
 
     // run it
     spawn(process.execPath, argv, { detached: true });
 
     // restore environment variables
-    unsetenv('RS_INSTALL_WD');
```

Comment:
RS_INSTALL_WD doesn't even exist, pretty sure that was a typo and RS_INITIAL_WD was intended. These get unset in the C++ code anyway, so I'm not sure that it matters